### DISCUSSION TITLE
Filter by Relation Field not working

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2985] [Q&A] [answered]
When you have time, please check filter by relation fields. Please take a look my capture

![CleanShot 2023-07-24 at 18 02 18@2x](https://github.com/pocketbase/pocketbase/assets/8098916/dd31adf5-ec77-4edf-906b-d91fa275fa19)
![CleanShot 2023-07-24 at 18 02 27@2x](https://github.com/pocketbase/pocketbase/assets/8098916/c70abf57-0ac5-4ca5-821f-c63e2276b1be)

**Comment by ganigeorgiev** [comment, marked as answer]
If you want a "at least one of" type of filter for multiple fields you'll need to prefix the operator with `?`, aka. `tags.name ?~ "for"`.

**Reply by hungcrush** [reply]
@ganigeorgiev I just want to say... Thank you so much~~~
